src/js/profile/mobile/widget/Calendar.js

```diff
@@ -169,43 +223,96 @@
 			* @protected
 			*/
 			prototype._onClick = function (event) {
-				var calendarElement = this.element.querySelector(".calendarView");
+				var calendarElement = this.element.querySelector(".calendarView"),
+					self = this;
 
 				if (event.srcElement.className === "calendarArrowRight") {
-					todayMonth = todayMonth + 1;
-					if (todayMonth === 13) {
-						todayMonth = 1;
-						todayYear = todayYear + 1;
-					}
-					prototype._deleteCalendar(calendarElement);
-					today = new Date(todayYear, todayMonth - 1);
-					prototype._buildCalendar(calendarElement);
+					self._moveMonth("right", calendarElement);
 				} else if (event.srcElement.className === "calendarArrowLeft") {
-					todayMonth = todayMonth - 1;
-					if (todayMonth === 0) {
-						todayMonth = 12;
-						todayYear = todayYear - 1;
-					}
-					prototype._deleteCalendar(calendarElement);
-					today = new Date(todayYear, todayMonth - 1);
-					prototype._buildCalendar(calendarElement);
+					self._moveMonth("left", calendarElement);
+				} else if (event.srcElement.className === "calendarArrowLeft nonSelectionArrow") {
```

Comment:
if you would have `self._ui.leftArrow` you could be compare like this:
`event.srcElement === self._ui.leftArrow && self._ui.leftArrow.classList.contain(...)